spec/models/dhs1171_pdf_spec.rb

```diff
@@ -45,8 +55,8 @@ def filled_in_values(file:)
     end
   end
 
-  def generated_pdf
-    @_generated_pdf ||= "tmp/dhs1171_test_output.pdf"
+  def new_pdf_filename
+    @_new_pdf_filename ||= "dhs1171_test_output.pdf"
```

Comment:
Oh no - this is just the filename which is passed into the constructor without a path. So this is correct.